README.md

```diff
@@ -59,6 +59,36 @@ For example, some changes in the Selenium binding could break the Appium client.
 > to keep compatible version combinations.
 
 
+### Quick migration guide from v4 to v5
+- This change affecs only for a user who speficies `keep_alive`, `direct_connection` and `strict_ssl` arguments for `webdriver.Remote`:
+    - Please use `AppiumClientConfig` as `client_config` arguemnt as below:
+        ```python
+        SERVER_URL_BASE = 'http://127.0.0.1:4723'
+        # before
+        driver = webdriver.Remote(
+            SERVER_URL_BASE,
+            options=UiAutomator2Options().load_capabilities(desired_caps),
+            direct_connection=True,
+            keep_alive=False,
+            strict_ssl=False
+        )
+
+        # after
+        from appium.webdriver.client_config import AppiumClientConfig
+        client_config = AppiumClientConfig(
+            remote_server_addr=SERVER_URL_BASE,
+            direct_connection=True,
+            keep_alive=False,
+            ignore_certificates=True,
+        )
+        driver = webdriver.Remote(
+            options=UiAutomator2Options().load_capabilities(desired_caps),
+            client_config=client_config
+        )
+        ```
+        - Note that you can keep using `webdriver.Remote(url, options=options, client_config=client_config)` format as well. Then, the `remote_server_addr` in `AppiumClientConfig` will prior than the `url` specified via `webdriver.Remote`
```

Comment:
Then, the ... -> In such case the `remote_server_addr` value of `AppiumClientConfig` would have priority over the `url` argument of `webdriver.Remote`